Jenkinsfile

```diff
@@ -1,4 +1,5 @@
 #!groovy
+
```

Comment:
please, revert

Reply:
I'm sorry Evgeny, I'm afraid I can't do that 
![image](https://user-images.githubusercontent.com/7278796/96126100-b1a72200-0efc-11eb-95ee-6c17d9075a8a.png)

It is funny because I've somehow managed to commit directly to @achetver repository via GitHub web UI. Unfortunately, it has no 'Revert' button on commits that aren't merge or squashed commit from PR.
So I kindly ask @achetver to execute the following command:
```
git revert 56e6ae1e4c31439ba0d4636fa76782c03bf30aca && git push
```

Reply:
Ok, done :)

Reply:
Thanks!